backend/app/controllers/tasks.js

```diff
@@ -427,7 +428,11 @@ function* updateCurrentStepId(req, insertedTaskId) {
                 .where(Workflow.productId.equals(req.body.productId))
                 .and(WorkflowStep.position.equals(0)))).id;
         } else {
-            // TODO: INBA-561
+            const priorStep = _.find(result, (step) => step.position === addedStep.position -1);
```

Comment:
Use either `addedStep.position-1` or `addedStep.position - 1`, preferably the latter. This gives me OCD night sweats.